Changelog.md

```diff
@@ -0,0 +1,36 @@
+## Release 0.1 (07/01/23)
+
+### Additions
```

Comment:
Could we go with the same headings as [keep a changelog](https://keepachangelog.com/en/1.0.0/)?

Reply:
done